.github/workflows/trigger-flatpak-update.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger Flatpak Headlamp version update
+
+# See https://github.com/flathub/io.kinvolk.Headlamp
+on:
```

Comment:
Adding a manual trigger here will be useful to debug when things fail. Apart from that looks good.

Reply:
Good point!

Reply:
Done.